### DISCUSSION TITLE
How do I find the function for `M-x terminal`?

### DISCUSSION THREAD

**Discussion by g-gundam** [lem-project/lem#1506] [Q&A] [answered]
I want to create a shorter keybinding for `M-x terminal`, but I can't find the function that `M-x terminal` invokes.  I even did `(apropos "terminal")` in the REPL to find some candidates, but I haven't been able to find it yet.  Is there a better way to find the fully qualified function name for a command?

**Comment by g-gundam** [comment]
I was [told](https://desuarchive.org/g/thread/102058204/#102163711) by some anon that it was `lem-terminal/terminal-mode::terminal`, but the general question still remains.  There's gotta be a better way to translate a known command name `terminal` and get `lem-terminal/terminal-mode::terminal`.

**Comment by cxxxr** [comment, marked as answer]
Yes, you can get it with this.
```
(lem:find-command "terminal")
```
